analyzer/compliance-analyzer/pom.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+ <project xmlns="http://maven.apache.org/POM/4.0.0"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+     <parent>
+         <artifactId>analyzer</artifactId>
+         <groupId>eu.fasten</groupId>
+         <version>0.0.1-SNAPSHOT</version>
+     </parent>
+     <modelVersion>4.0.0</modelVersion>
+
+      <groupId>eu.fasten.analyzer</groupId>
+     <artifactId>compliance-analyzer-plugin</artifactId>
+     <version>0.0.1-SNAPSHOT</version>
+     <packaging>jar</packaging>
+
+      <name>compliance-analyzer-plugin</name>
+
+      <dependencies>
+         <dependency>
+            <groupId>info.picocli</groupId>
+            <artifactId>picocli</artifactId>
+            <version>4.0.4</version>
+         </dependency>
+         <dependency>
+            <groupId>eu.fasten</groupId>
+            <artifactId>core</artifactId>
+            <version>0.0.1-SNAPSHOT</version>
+         </dependency>
+         <dependency>
+            <groupId>eu.fasten</groupId>
+            <artifactId>server</artifactId>
+            <version>0.0.1-SNAPSHOT</version>
+            <!--<scope>provided</scope>-->
+         </dependency>
+         <!--NB! Uncomment the following 3 dependencies if you want to run Main class-->
+
+         <!-- . -->
```

Comment:
Can be removed

Reply:
Done in https://github.com/fasten-project/fasten/commit/7975246450da52793713310dc6123c14c876daf5.